sources/packages/web/src/services/http/dto/Institution-user.dto.ts

```diff
@@ -0,0 +1,119 @@
+import { AddressAPIOutDTO } from "@/services/http/dto";
+
+export interface InstitutionUserAuthTypeAPIOutDTO {
+  role: string;
+  type: string;
+}
+
+export interface InstitutionLocationAuthDataAPIOutDTO {
+  address: AddressAPIOutDTO;
+}
+
+export interface InstitutionAuthLocationAPIOutDTO {
+  id: number;
+  name: string;
+  data: InstitutionLocationAuthDataAPIOutDTO;
+}
+
+export interface InstitutionUserAuthAPIOutDTO {
+  id?: number;
+  authType: InstitutionUserAuthTypeAPIOutDTO;
+  location?: InstitutionAuthLocationAPIOutDTO;
+}
+
+export interface InstitutionUserSummaryAPIOutDTO {
+  id: number;
+  email: string;
+  firstName: string;
+  lastName: string;
+  userName: string;
+  isActive: boolean;
+  userFullName: string;
+}
+
+export interface InstitutionUserAPIOutDTO {
+  id: number;
+  user: InstitutionUserSummaryAPIOutDTO;
+  authorizations: InstitutionUserAuthAPIOutDTO[];
+}
+
+export interface UserPermissionAPIInDTO {
+  locationId?: number;
+  userType?: string;
+  userRole?: string;
+}
+
+/**
+ * Associates a new user from BCeID with an institution
+ * associating also the authorizations.
+ */
+export interface CreateInstitutionUserAPIInDTO {
+  /**
+   * User BCeID id from BCeID Web Service (e.g. SomeUserName) that will have its
+   * data retrieved to be created on SIMS.
+   */
+  bceidUserId: string;
+  /**
+   * Permissions to be associated with the new user.
+   */
+  permissions: UserPermissionAPIInDTO[];
+}
+
+/**
+ * Update an existing user association with an institution
+ * changing the authorizations.
+ */
+export type UpdateInstitutionUserAPIInDTO = Omit<
+  CreateInstitutionUserAPIInDTO,
```

Comment:
The CreateInstitutionUserAPIInDTO does not have userId? did u mean bceidUserId?

Reply:
Yes, `bceidUserId`. I will change it.